metagraph/src/common/vectors/bitmap_builder.hpp

```diff
@@ -75,4 +78,35 @@ class bitmap_builder_set : public bitmap_builder {
 };
 
 
+class bitmap_builder_set_disk : public bitmap_builder {
+  public:
+    bitmap_builder_set_disk(uint64_t size,
+                            size_t num_threads,
+                            uint64_t buffer_size,
+                            const std::string &swap_dir,
+                            size_t max_disk_space_bytes = -1,
+                            size_t merge_count = 4)
```

Comment:
It's not clear what `merge_count` is supposed to be. Add a comment

Reply:
I'll just make it a constant. I doubt anyone will change this parameter.